Controller/Callback/Offsite.php

```diff
@@ -66,7 +67,7 @@ public function execute()
             return $this->redirect(self::PATH_CART);
         }
 
-        if ($order->getState() !== Order::STATE_PENDING_PAYMENT) {
+        if (! in_array($order->getState(), [ Order::STATE_PENDING_PAYMENT, Order::STATE_PROCESSING ])) {
```

Comment:
@guzzilar do you have any idea.. why it could send me here PROCESSING, not PENDING like all other offsite payment method? it is Callback.
:(

Reply:
both alipay, and truemoney returns status from omise gateway `"status": "pending"` (found inside response json)
but for some reason, the mag 2 framework returns in `truemoney` case status `processing`, and `alipay` - `pending`